### ISSUE TITLE
comemu here/inside does not go to current directory

### ISSUE THREAD

**Issue by Mete0** [cmderdev/cmder#1024]
Whenever I use the context menu and select comemu here or inside, the console opens in the default directory. Here are my settings:

![image](https://cloud.githubusercontent.com/assets/13852752/16604256/69cb9640-42f1-11e6-8872-82d64624e62e.png)

**Comment by Mr-Binary** [comment]
add "shell" in the configuration textblock, it will behave like powershell, but now can work without bug

**Comment by rainyday** [comment]
The above link is broken. Been trying to find a fix for this for at least a year.

**Comment by Mr-Binary** [comment]
it's weird. try to copy the link text and open it as a URL, it works.

**Comment by Stanzilla** [comment]
Fixed it for you

**Comment by CodingMonkeyXYZ** [comment]
You're supposed to be able to do this by adding the -here argument to the command under the 'ComEnu Here' settings on the Integration menu but this doesn't override the settings for the tasks stored under the startup menu, tasks. So in other words, Cmder will always start in the directory specified under tasks (user's profile dir) and not 'here', which is clearly a bug.

**Comment by chrisdarth** [comment]
I did import the .reg-file in the Stack Overflow answer (this may have had no impact as I noticed the entries already seemed to exist). I'm now seeing this behavior:

I use PowerShell and have this as my _Default task for new console_

`PowerShell -ExecutionPolicy Bypass -NoLogo -NoProfile -NoExit -Command "Invoke-Expression '. ''%ConEmuDir%\..\profile.ps1'''" -new_console:d:"%USERPROFILE%"`

If i type `Cmder` in the Windows Explorer address bar, it launches a new instance, but in my `%USERPROFILE%` directory. Which I guess is what I should expect. All tasks are started in this directory.

If I change the task to this

`PowerShell -ExecutionPolicy Bypass -NoLogo -NoProfile -NoExit -Command "Invoke-Expression '. ''%ConEmuDir%\..\profile.ps1'''"` 

where I remove the `new_console` parameter, typing `Cmder` in Explorer address bar works, but launching Cmder from my shortcut starts in `C:\Program Files\Cmder` (where I have the cmder.exe).

I checked the shortcut, and see that it's set up with a startup directory that can be changed to whatever I need (I changed this to `%USERPROFILE%`).
![Imgur](http://i.imgur.com/zNgINy1.png)

It now seems to work as how I want it. Only thing which puzzles me is how Cmder knows to start in my home directory if I run Win+R -> `cmder`.
